src/filter/distort0r/distort0r.c

```diff
@@ -66,6 +69,16 @@ void f0r_get_param_info(f0r_param_info_t* info, int param_index)
       info->type = F0R_PARAM_DOUBLE;
       info->explanation = "The frequency of the plasma signal";
       break;
+    case 2:
+      info->name = "Mode";
+      info->type = F0R_PARAM_BOOL;
+      info->explanation = "\'Time Based\' or \'Adjustable Velocity\'";
```

Comment:
>Mode with this explanation does not make it clear...

Firstly, i thought to do a "Use Velocity.", then i make a choice (wrong one), thinking of future, when more Mode could be implemented.... but KISS is the way.